css/amp-story-player-iframe.css

```diff
@@ -41,6 +41,7 @@ main {
 
 .i-amphtml-story-player-loaded iframe {
   opacity: 1;
+  transition: transform 250ms cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
How does it feel? It looks a bit slow for a horizontal animation on a phone, but I trust your judgement here :)

Reply:
Cut it to 200ms, it feels better.